azkaban-common/src/test/java/azkaban/executor/MockExecutorLoader.java

```diff
@@ -303,6 +303,15 @@ public Executor addExecutor(String host, int port)
   }
 
   @Override
+  public void removeExecutor(String host, int port) throws ExecutorManagerException {
```

Comment:
nit:
I prefer using mockito rather than using a complete mocked class. You can spy on methods and validate calls etc. But this is up to you.